### ISSUE TITLE
WiFiClientSecure fails just by having the Audio object created

### ISSUE THREAD

**Issue by codegrue** [schreibfaul1/ESP32-audioI2S#679]
Doing nothing more than having this line:

`Audio audio;`

Causes all SSL API calls to fail with either memory issue or certificate verification failure:

`[start_ssl_client():273]: (-9984) X509 - Certificate verification failed, e.g. CRL, CA or signature check failed`

I suspected the heap getting too small, but I still have 40000 bytes. Are there any known interactions between ESP32-audioI2S and WifiClientSecure?

**Comment by codegrue** [comment]
I did confirm that this was a memory related issue. I was unable to get Audio to use PSRAM, and WifiClientSecure also was using main heap. I did figure out how to solve the second item, but I still suspect Audio won't use it because implementing it as a global variable, it get's created before the ESP32 PSRAM can be initialized in app_main(void). I did end up moving to a different audio stack before I was able to solve it, so I'm not sure if the technique would work where you create Audio as a static in a function (on first use essentially) rather than as a global.
